### ISSUE TITLE
NVDA reports the status of the flight mode twice in the Explorer

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#11849]
### Steps to reproduce:
I have to say that only a few laptop have a key to switch the flight mode switch.
My laptop model is Dell Latitude 7490, which happens to have this feature.
Press Win + M to go to the desktop, then press FN + PrintScreen.

### Actual behavior:
Report "Enable airplane mode"("Disable airplane mode") twice in a row.
I suspect that the cause of this problem is the same as [9466](https://github.com/nvaccess/nvda/pull/9466).

### Expected behavior:
Report "Enable Airplane Mode"("Disable airplane mode") only once

#### NVDA installed/portable/running from source:
Installed and portable

#### NVDA version:
2020.3 And
 Alpha 21386

#### Windows version:
Windows version: Windows 10 20H2 (64-bit) build 19042.630

#### Name and version of other software in use when reproducing the issue:
none

#### Other information about your system:
none

#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
NVDA2020.2; NVDA2020.1Have the same problem

#### If addons are disabled, is your problem still occuring?
yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes.

**Comment by josephsl** [comment]
Hi, I suspect duplicate UIA notification event. With Windows 10 App Essentials add-on installed, can you restart NVDA with debug logging enabled, perform airplane mode toggle command, and attach the resulting log? If it is what I think it is, I’ll write a workaround in Windows 10 App Essentials add-on in December. CC @LeonardDeR

**Comment by cary-rowen** [comment]
Hi joseph,
I have attached a debug log.
I can still reproduce the problem without installing any plug-ins, so I hope NVDA can fix it.
Of course, thank you very much for your efforts.

[NVDA_Log.log](https://github.com/nvaccess/nvda/files/5578007/NVDA_Log.log)

**Comment by josephsl** [comment]
Hi, at least this confirms the issue, but that’s not the data I need. Please install Windows 10 App Essentials add-on and then attach the new log. Thanks.

**Comment by cary-rowen** [comment]
Hi, joseph.
Sorry, my log debugging level setting is inappropriate.
I have re-uploaded the new log.
I think this is what you need.
Thanks.
[NVDA_Log_Debug.log](https://github.com/nvaccess/nvda/files/5578036/NVDA_Log_Debug.log)



@josephsl

**Comment by josephsl** [comment]
Hi, yes, this is the log I was looking for, and yes, UIA notification event handler should be tweaked. I’ll get to it this weekend and come up with a workaround via Windows 10 App Essentials add-on by end of November. Thanks.

**Comment by josephsl** [comment]
Hi,

It turns out this might not be a UIA notification event handler issue - based on comparing logs (between my debug log and the most recent debug log attached), later airplane mode toggle announcement should be heard as speech is cut off.

What speech synthesizer are you using? Can you test it with a different speech synthesizer and see if airplane mode toggle is announced only once? If yes, this is a synthesizer problem, not really a UIA event handler issue.

Also, to help us get a better picture, can people speaking different languages and speech synthesizers (and braille displays) try replicating this problem?

Thanks.

**Comment by CyrilleB79** [comment]
Issue confirmed on French Windows with all synth. Tested the following synth:
* Windows OneCore
* eSpeak
* IBMTTS
Config:
* Windows 10 2004 (French)
* NVDA 2020.3
* Computer HP

The announcement is only made in Windows Explorer, not in other apps, as for volume announcement.

**Comment by josephsl** [comment]
Hi, so you are saying that NVDA announces airplane mode toggle messages in French twice? This is a completely different issue than a later note about this being announced only in File Explorer – that issue was covered earlier and a PR exists (if you are implying that this should be announced everywhere, then that one has nothing to do with this one). Thanks.

**Comment by Brian1Gaff** [comment]
Just an aside, but even in windows 7, there are instances where things get 
multiple announced. Try creating a new folder in explorer in Windows 7.

I thought this kind of issue has been a bit of a problem with nvda for may 
years.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by CyrilleB79** [comment]
> Hi, so you are saying that NVDA announces airplane mode toggle messages in French twice? This is a completely different issue than a later note about this being announced only in File Explorer – that issue was covered earlier and a PR exists (if you are implying that this should be announced everywhere, then that one has nothing to do with this one). Thanks.

Well, I do not know if I understand well your comment and if I made myself well understood.

So to summarize, in French and when I am in Explorer or on the Desktop, I have the airplane mode status reported 2 times when I press Fn+F12 (the airplane mode shortcut on my keyboard). I would have expected this status to be reported only once.
HTH

**Comment by josephsl** [comment]
Hi, yes, that’s the issue we’re covering here. Is this reproducible when using different speech synthesizers? Thanks.

**Comment by CyrilleB79** [comment]
> Hi, yes, that’s the issue we’re covering here. Is this reproducible when using different speech synthesizers? Thanks.

Yes, as written in my previous message, it is reproducible with all 3 tested synthesizers: Windows OneCore, eSpeak and IBMTTS.

**Comment by josephsl** [comment]
Hi, got it. @LeonardDer, any thoughts? Thanks.

**Comment by cary-rowen** [comment]
Hi,

It is newly discovered that although adjusting the volume in the explorer has not been reported twice, it can be found from the reading viewer that NVDA has output the same content twice.
@josephsl

**Comment by Adriani90** [comment]
@cary-rowen how is this now in the last Windows 10 update?

**Comment by cary-rowen** [comment]
Unsurprisingly, nothing changed.

**Comment by Adriani90** [comment]
@cary-rowen did you report this to Microsoft already?

**Comment by cary-rowen** [comment]
No, I did not report it to Microsoft
